torch/_inductor/scheduler.py

```diff
@@ -408,6 +408,11 @@ def decide_inplace_update(self) -> None:
         }
 
         ordered_reads = sorted(self.read_writes.reads, key=lambda x: x.name)
+        # NOTE remove V.graph.removed_operations once deps issue is fixed
+        inconsequential_nodes = (self.ancestors - {self.get_name()}) \
```

Comment:
I'm not sure if self can be in ancestors at this point in the IR, open to feedback from reviews.